README.md

```diff
@@ -1,2 +1,109 @@
-# terra-graphs
-A lightweight, JavaScript visualization API built using D3. Includes a vanilla javascript and react implementation.
+<!-- Logo -->
+<p align="center">
+  <img height="128" width="128" src="https://github.com/cerner/terra-graphs/raw/main/terra.png" />
```

Comment:
I'd prefer to start aligning with the terra icon. The image is broken for the deployment because it's referencing the main branch for the image, once this PR is merged down the image will be found in the right location.